Source/WebCore/css/parser/CSSSelectorParser.h

```diff
@@ -45,22 +44,6 @@ class CSSSelectorList;
 class StyleSheetContents;
```

Comment:
forward declare it here.
struct CSSSelectorParserContext;

Reply:
Unfortunately I can't because of the `const CSSSelectorParserContext m_context;` member, it will complain about incomplete type